broker/pom.xml

```diff
@@ -50,6 +51,10 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>rocketmq-acl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
```

Comment:
Here,why is broker dependent on the mqtt module.I remenber the snode module has already been dependent on the mqtt module?

Reply:
I think the relationship for the dependency of mqtt module is complex. @Aaron-He @xiang

Reply:
1. MQTTProcessor in Enode module uses MqttConstant in mqtt module. 
2. Snode module uses DefaultMqttMessageProcessor in mqtt module.